TTS/utils/audio.py

```diff
@@ -150,12 +150,15 @@ def normalize(self, S):
         if self.signal_norm:
             # mean-var scaling
             if hasattr(self, 'mel_scaler'):
-                if S.shape[0] == self.num_mels:
+                signal_shape = S.shape[0]
+                if signal_shape == self.num_mels:
                     return self.mel_scaler.transform(S.T).T
-                elif S.shape[0] == self.fft_size / 2:
+                elif signal_shape == ((self.fft_size // 2) + 1):
```

Comment:
why do you think this is false ? I don't think ```((self.fft_size // 2) + 1)``` makes any difference.

Reply:
@erogol Two reasons why I think the existing code is wrong:
1. When using the CheckSpectograms notebook, if `signal_norm` is set to `true`, it is not possible to load stats without changing this line. It will always result in the error "Mean-Var stats shape ({signal_shape}) does not match the given feature dimensions."
2. Both https://github.com/coqui-ai/TTS/blob/main/TTS/tts/utils/generic_utils.py#L55 and https://github.com/coqui-ai/TTS/blob/main/tests/test_loader.py#L80 use `fft_size / 2 + 1`.